src/metric/definition.go

```diff
@@ -565,6 +565,42 @@ var EtcdQueries = []prometheus.Query{
 
 // KSMSpecs are the metric specifications we want to collect from KSM.
 var KSMSpecs = definition.SpecGroups{
+	"persistentvolume": {
+		// kube_persistentvolume_created is marked as an experimental metric, so we instead use the namespace and name
```

Comment:
Why do we scrape an experimental metric that we do not use? This metric is not reported in Kubernetes v1.19 and v1.20. The metric was only added on May 23 2022.

Consider removing the variable since static integration tests are failing for this metric

Reply:
We do use it for the `createdAt` metric. It is experimental, but seems very likely to change. To fix the static tests, let's make the `createdAt` metric below optional.

Reply:
Ok